lambdas/meetupcom/config.js

```diff
@@ -21,7 +29,13 @@ const groupsParams = convert({
 const eventsApi = slug => `https://api.meetup.com/${slug}/events`;
 const eventsParams = convert({
   "photo-host": "public",
-  status: "cancelled,past,proposed,suggested,upcoming",
+  status: [
+    "cancelled",
+    "past",
+    "proposed",
```

Comment:
NAB: How does `proposed` and `suggested` differ?

Reply:
@apoclyps Good question - they're provided by meetup.com, but the docs don't describe the differences
https://www.meetup.com/meetup_api/docs/:urlname/events/#listparams

The only thing mentioned is `proposed`

> Creating an event without a time will result in an event with a "proposed" status.